project/Dependencies.scala

```diff
@@ -5,7 +5,7 @@ import sbt._
 object Dependencies {
   val identityLibVersion = "4.31"
   val awsVersion = "1.12.780"
-  val awsSdk2Version = "2.26.31"
+  val awsSdk2Version = "2.30.37"
```

Comment:
This is good and will boost Netty almost everywhere - almost to the point where you don't need to specify the Netty update yourself, but it looks like `netty-transport-native-epoll` is one library that Play depends on, and the AWS SDK doesn't, so updating the AWS SDK itself isn't enough to get all Netty libraries in sync....

This problem - of keeping different subartifacts of a library in sync - is one I think that Maven in general addresses with BOM, and there are the some attempts to bring this functionality to sbt - like https://github.com/heremaps/here-sbt-bom - but I haven't tried them myself yet.